Test started to fail on cron jobs:    OSError: Unable to find libfuse

```
py3 run-test: commands[1] | coverage run -m pytest datalad_fuse
ImportError while loading conftest '/home/runner/work/datalad-fuse/datalad-fuse/datalad_fuse/tests/conftest.py'.
datalad_fuse/__init__.py:18: in <module>
    from fuse import FUSE
.tox/py3/lib/python3.7/site-packages/fuse.py:115: in <module>
    raise EnvironmentError('Unable to find libfuse')
E   OSError: Unable to find libfuse
ERROR: InvocationError for command /home/runner/work/datalad-fuse/datalad-fuse/.tox/py3/bin/coverage run -m pytest datalad_fuse (exited with code 4)
```

I think there might have been ubuntu base change on github actions recently. Started to fail 3 days ago according to https://github.com/datalad/datalad-fuse/actions

--- a/datalad_fuse/__init__.py
+++ b/datalad_fuse/__init__.py
@@ -15,10 +15,8 @@ from datalad.interface.results import get_status_dict
 from datalad.interface.utils import eval_results
 from datalad.support.constraints import EnsureNone
 from datalad.support.param import Parameter
-from fuse import FUSE
 
 from ._version import get_versions
-from .fuse_ import DataLadFUSE
 
 # Defines a datalad command suite.
 # This variable must be bound as a setuptools entrypoint
@@ -107,6 +105,10 @@ class FuseFS(Interface):
         mode_transparent: bool = False,
         allow_other: bool = False,
     ) -> Iterator[Dict[str, Any]]:
+        from fuse import FUSE
+
+        from .fuse_ import DataLadFUSE
+
         if not foreground:
             yield get_status_dict(
                 action="fusefs",

